crates/taiko/evm/src/execute.rs

```diff
@@ -338,23 +458,214 @@ where
     }
 }
 
+/// A taiko block executor that uses a [`BlockExecutionStrategy`] to
+/// execute blocks.
+#[allow(missing_debug_implementations, dead_code)]
```

Comment:
```suggestion
#[expect(missing_debug_implementations, dead_code)]
```
Is it impossible to derive debug in here?